e2e/config.yaml.sample

```diff
@@ -1,11 +1,35 @@
 ---
+# Configuration file for RamenDR E2E testing on Regional DR.
+
+# Name of the channel used for the Channel CR (utilized by Subscription CRs).
 channelname: "ramen-gitops"
+
+# Namespace where the channel CR will be created.
 channelnamespace: "ramen-samples"
+
+# Git repository URL containing application manifests to be
+# deployed on the clusters.
 giturl: "https://github.com/RamenDR/ocm-ramen-samples.git"
+
+# List of PVC specifications for workloads.
+# These define storage configurations, such as 'storageClassName' and
+# 'accessModes', and are used to kustomize workloads.
 pvcspecs:
   - name: rbd
     storageclassname: rook-ceph-block
     accessmodes: ReadWriteOnce
   - name: cephfs
     storageclassname: rook-cephfs
     accessmodes: ReadWriteMany
+
+# Sample cluster configurations (commented out by default):
+# Uncomment and edit the following lines to provide the kubeconfig paths
+# for your test clusters.
+
```

Comment:
Lets remove the blank line - the  all the other comments are right above the yaml item, so this should be right above the "clusters:".

Reply:
Removed